docusaurus.config.js

```diff
@@ -18,8 +18,8 @@ const config = {
 
   // GitHub pages deployment config.
   // If you aren't using GitHub pages, you don't need these.
-  organizationName: 'facebook', // Usually your GitHub org/user name.
-  projectName: 'docusaurus', // Usually your repo name.
+  organizationName: 'responsivedev', // Usually your GitHub org/user name.
```

Comment:
I'm pretty sure we don't use this but it's probably best to have things set correctly in case we ever do in the future